src/app/clusters/valve-configuration-and-control-server/valve-configuration-and-control-server.cpp

```diff
@@ -223,12 +190,12 @@ static void onValveConfigurationAndControlTick(System::Layer * systemLayer, void
 
     if (rDuration.Value() > 0)
     {
-        SetRemainingDuration(ep, DataModel::MakeNullable<uint32_t>(--rDuration.Value()));
+        ValveConfigurationAndControl::SetRemainingDuration(ep, DataModel::MakeNullable<uint32_t>(--rDuration.Value()));
```

Comment:
why is the logic here using 0 for null when the value is a nullable already? Why is null not null?
Some comments explaining would be nice.